public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/styles.scss

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
In order to display the embeddable correctly we should add styling to the vis similar to this:https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/commit/317b33676af31e67df2120804b85d8a146fa5be9#diff-a9719e85099dbb53d9d031c8ba22bdfcae1606179e6debf74e39b6758311edc3R1

Reply:
updated in the new revision

Reply:
I don't see this style, am I missing it somewhere?